cmd/cloudFoundryDeploy.go

```diff
@@ -668,21 +672,101 @@ func deployMta(config *cloudFoundryDeployOptions, mtarFilePath string, command c
 		cfDeployParams = append(cfDeployParams, deployParams...)
 	}
 
-	cfDeployParams = append(cfDeployParams, handleMtaExtensionDescriptors(config.MtaExtensionDescriptor)...)
+	extFileParams, extFiles := handleMtaExtensionDescriptors(config.MtaExtensionDescriptor)
 
-	return cfDeploy(config, cfDeployParams, nil, nil, command)
+	for _, extFile := range extFiles {
+		_, err := fileUtils.Copy(extFile, extFile+".original")
+		if err != nil {
+			return err
```

Comment:
Would wrap the error here. The error will only say that copying the file failed, but not why or in which context it was copied.

Reply:
done